src/main/java/remi/storage/Storage.java

```diff
@@ -0,0 +1,105 @@
+package remi.storage;
+
+import remi.model.Deadline;
+import remi.model.Event;
+import remi.model.StoredTaskList;
+import remi.model.ToDo;
+
+import java.io.*;
```

Comment:
The coding standard is strictly against wildcard imports. I don't see any reason to import everything too. Perhaps you should replace it with named imports instead? And maybe you should try configuring IntelliJ not to merge your name imports into wildcard import.